test/XRP/xrp-client-integration-test.ts

```diff
@@ -115,20 +115,35 @@ describe('Xpring JS XRPClient Integration Tests', function(): void {
     assert.equal(doesExist, true)
   })
 
-  it('Check if Account Exists - Web Shim', async function(): Promise<void> {
+  it('Check if Account Exists - true - Web Shim', async function(): Promise<
+    void
+  > {
     this.timeout(timeoutMs)
 
     const doesExist = await xpringWebClient.accountExists(recipientAddress)
     assert.equal(doesExist, true)
   })
 
-  it('Check if Account Exists - rippled', async function(): Promise<void> {
+  it('Check if Account Exists - true - rippled', async function(): Promise<
+    void
+  > {
     this.timeout(timeoutMs)
 
     const doesExist = await xrpClient.accountExists(recipientAddress)
+
     assert.equal(doesExist, true)
   })
 
+  it('Check if Account Exists - false - rippled', async function(): Promise<
+    void
+  > {
+    this.timeout(timeoutMs)
+
+    const coinbaseMainnet = 'XVYUQ3SdUcVnaTNVanDYo1NamrUukPUPeoGMnmvkEExbtrj' // valid address but should NOT show up on testnet
```

Comment:
Nice. Whose address is this?

Reply:
it's the coinbase mainnet! so by using it on Testnet we know that it's 1) valid in format and 2) nonexistant